### PR TITLE
Add io.otsaloma.nfoview

### PR DESCRIPTION
**Pull request by otsaloma**
https://otsaloma.io/nfoview/
https://github.com/otsaloma/nfoview

### CODE REVIEW

File: io.otsaloma.nfoview.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+app-id: io.otsaloma.nfoview
+runtime: org.gnome.Platform
+runtime-version: 3.30
```
Comment (x80486) [inline io.otsaloma.nfoview.yml]: Should be `runtime-version: "3.30"`...like _stringified_.

File: io.otsaloma.nfoview.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+sdk: org.gnome.Sdk
+command: nfoview
+rename-appdata-file: nfoview.appdata.xml
```
Comment (bilelmoussaoui) [inline io.otsaloma.nfoview.yml]: Should be renamed upstream. Same for the icon, desktop file. 
See the specs:
https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s02.html
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

File: io.otsaloma.nfoview.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+rename-desktop-file: nfoview.desktop
+rename-icon: nfoview
+copy-icon: true
+finish-args:
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --filesystem=host
```
Comment (bilelmoussaoui) [inline io.otsaloma.nfoview.yml]: You should use NativeFileChooser (which uses the portal) and remove the host file permission.

File: io.otsaloma.nfoview.yml (hunk 1 above)
Comment (otsaloma) [reply io.otsaloma.nfoview.yml]: Done, same for the git tag version.

File: io.otsaloma.nfoview.yml (hunk 3 above)
Comment (otsaloma) [reply io.otsaloma.nfoview.yml]: Assuming you mean [GtkFileChooserNative](https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html), that seems to require GTK+ 3.20 and I'd rather not bump the GTK+ requirement that far yet on account of native distro packages.

Comment (barthalion) [reply io.otsaloma.nfoview.yml]: I'm hardly a real programmer but I find it hard to believe it's impossible to support both cases.

Comment (barthalion) [reply io.otsaloma.nfoview.yml]: From major distributions, only Slackware 14.2 and Debian oldstable ships gtk3 older than gtk 3.20.

**Comment by barthalion** [comment]
bot, build io.otsaloma.nfoview
testbot, build io.otsaloma.nfoview

File: io.otsaloma.nfoview.yml (hunk 3 above)
Comment (otsaloma) [reply io.otsaloma.nfoview.yml]: It is actually likely that I can support both. Generally I don't like that pattern, but with this app the file dialog use is such simple, that it's likely doable. I'll look into it.

Comment (otsaloma) [reply io.otsaloma.nfoview.yml]: I just released 1.26, which uses Gtk.FileChooserNative when available and updated this manifest accordingly. I also removed dconf permissions, which I had only added on account the previous direct use of a file chooser dialog, nothing in nfoview itself uses dconf.

https://github.com/otsaloma/nfoview/commit/bad552bb3fb5136fc4d68959f2d8920eb096fe29

**Comment by barthalion** [comment]
bot, build io.otsaloma.nfoview
testbot, build io.otsaloma.nfoview

File: io.otsaloma.nfoview.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+runtime-version: "3.30"
+sdk: org.gnome.Sdk
+command: nfoview
+rename-appdata-file: nfoview.appdata.xml
+rename-desktop-file: nfoview.desktop
+rename-icon: nfoview
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --share=network
```
Comment (barthalion) [inline io.otsaloma.nfoview.yml]: What does it network access for?
Also given it uses GTK3, it should also work on Wayland.

Comment (otsaloma) [reply io.otsaloma.nfoview.yml]: Indeed, network isn't used, removed now and Wayland added.

**Comment by barthalion** [comment]
bot, build io.otsaloma.nfoview
testbot, build io.otsaloma.nfoview

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/io.otsaloma.nfoview

Thanks!

**Pull request closed without merging**